useful_scripts/install_dependencies.sh

```diff
@@ -0,0 +1,21 @@
+# This script installs the necessary dependencies on your computer so that you 
+# can set up Postgres in a Docker container and run the backend on your machine.
+
+
+echo "WARNING: This script requires root access to install dependencies."
+
+read -p "Do you still want to continue? [Y/n] " yn
+
+if [ $yn = "Y" ] || [ $yn = "y" ]; then
+	sudo apt install gdal-bin libgdal-dev
+	sudo apt install python3-gdal
+	sudo apt install binutils libproj-dev
```

Comment:
I deliberately avoided installing any "default" Python binaries, as I'm using virtual environments. Plus, I'm working on both Macs and Windows computers, and only Windows is capable of executing `apt` (under WSL). I know I don't have to use this script, but it would be better if it asked me whether I want to install Python.